File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TableViews_ViewModelMyElementsRow.cpp

#include "TableViews_ViewModelMyElementsRow.h"
#include <string>

namespace widgetassertions
{
  bool TableViews_ViewModelMyElementsRow::getIsIncludeCheckBoxChecked( ) 
  {
    return this->isIncludeCheckBoxChecked;
  }
  std::string TableViews_ViewModelMyElementsRow::getNameLabelText( ) 
  {
    return this->NameLabelText;
  }
  std::string TableViews_ViewModelMyElementsRow::getDescriptionLabelText( ) 
  {
    return this->DescriptionLabelText;
  }
  std::string TableViews_ViewModelMyElementsRow::getInfoIconImageName( ) 
  {
    return this->InfoIconImageName;
  }
  int TableViews_ViewModelMyElementsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TableViews_ViewModelMyElementsRow.cs
/// <filename>
///     TableViews_ViewModelMyElementsRow.cs
/// </filename>
namespace widgetassertions
{
    public class TableViews_ViewModelMyElementsRow
    {
        private bool isIncludeCheckBoxChecked;

        public virtual bool getIsIncludeCheckBoxChecked()
        {
            return this.isIncludeCheckBoxChecked;
        }

        private string NameLabelText;

        public virtual string getNameLabelText()
        {
            return this.NameLabelText;
        }

        private string DescriptionLabelText;

        public virtual string getDescriptionLabelText()
        {
            return this.DescriptionLabelText;
        }

        private string InfoIconImageName;

        public virtual string getInfoIconImageName()
        {
            return this.InfoIconImageName;
        }

        private int RowIndex;

        public virtual int getRowIndex()
        {
            return this.RowIndex;
        }
    }
}
